pytestqt/qtbot.py

```diff
@@ -399,8 +409,7 @@ def waitSignals(
         if order not in ["none", "simple", "strict"]:
             raise ValueError("order has to be set to 'none', 'simple' or 'strict'")
 
-        if raising is None:
-            raising = self._request.config.getini("qt_wait_signal_raising")
+        raising = self._should_raise(raising)
```

Comment:
Just noticed that I actually fixed another bug here :laughing: `qtbot.waitSignals()` didn't call `_parse_ini_boolean` like in other places, so `raising` was `""` (so false) when there is no entry for it in the .ini file. This actually ended up hiding a failing test in qutebrowser!